drakrun/drakrun/parse_xl_info.py

```diff
@@ -0,0 +1,30 @@
+import subprocess
+
+
+def get_xl_info():
+    xl_info_out = subprocess.check_output('xl info', shell=True).decode('utf-8', 'replace')
+    xl_info_lines = xl_info_out.strip().split('\n')
+
+    cfg = {}
+
+    for line in xl_info_lines:
+        k, v = line.split(':', 1)
+        k, v = k.strip(), v.strip()
+        cfg[k] = v
+
+    return cfg
+
+
+def get_xen_commandline(parsed_xl_info):
+    opts = parsed_xl_info['xen_commandline'].split(' ')
```

Comment:
This will easily break when someone inserts a tab or two spaces between command line parameters

Reply:
I've added gate against this:

```python
        if not opt.strip():
            continue
```